.github/actions/codacy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:11.0.1-jdk-slim-stretch
+
+COPY entrypoint.sh /bin/entrypoint.sh
+RUN chmod +x /bin/entrypoint.sh
+
+RUN apt-get update \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=5020686)